log/config.go

```diff
@@ -345,6 +347,20 @@ func Configure(options *Options) error {
 		}
 	}
 
+	if options.teeToUDSServer {
+		// build uds core.
+		core = teeToUDSServer(core, options.udsSocketAddress,
+			options.udsServerPath)
```

Comment:
nit: do we need to wrap these lines (here and below)? it is hard to tell in github, but if not a linter violation, let's make this one line.

Reply:
Done.